etc/wazo-agid/config.yml

```diff
@@ -6,46 +6,46 @@
 # should be.
 extra_config_files: /etc/wazo-agid/conf.d/
 
-# Owned of the process
+# Owner of the process
 user: wazo-agid
 
-# Change the log level to debug level.
+# Change the log level to debug.
 debug: false
 
 # Log file.
 logfile: /var/log/wazo-agid.log
 
-# Database connection informations.
+# Database connection settings
 db_uri: postgresql://asterisk:proformatique@localhost/asterisk?application_name=wazo-agid
 
-# AGI server listening informations.
+# AGI server settings
 listen_address: 127.0.0.1
 listen_port: 4573
 
-# wazo-agentd connection informations.
+# wazo-agentd connection settings
```

Comment:
I just wanted to fix the spelling error ("information" since its uncountable and thus should not have an "s"), but I figured "settings" would make more sense. So, it can't be fixed unless its fixed everywhere?

Reply:
Sure it can be fixed, but it would be very preferable to fix it everywhere too :)

Reply:
Oh for sure, but I have to remove the fix here and make another ticket and PR?

Reply:
Ok! The only issue will be finding them all 😅